### DISCUSSION TITLE
Extending hover styling options (Migrated)

### DISCUSSION THREAD

**Discussion by enkota** [elementor/elementor#30211] [Feature Request]
> [!NOTE]
> ### Moderation Edit: 
> We are merging all `:hover` styles-related requests with this one for easier referencing in the future. 

I was thinking of some ways the hover customizations could be extended.

**Images**
Hover Opacity - On hover the image opacity could be changed. 
Hover Scale - Size/Padding could be changed too.
Hover Overlay - On hover the caption could appear or just dim the image with a customisable colour.

**Sections/Columns**
Hover styling -  Could be applied to the entire section/column to further customisation options.

**Global**
Transition options - Linear curve etc/time/easing.
Scale - Background overlay - Scale on hover.

Just an idea :)

**Comment by drawcard** [comment]
Cool idea. If you need to do this quickly you could use the Custom CSS field in Elementor Pro and add to the existing transition effects.

**Comment by netgain** [comment]
The main thing I wanted to change when I used it was the speed of the hover effects. Even on the fast setting, it was much too slow. Rather than "slow / medium / fast", a numeric value would be much better. Or, to be fancy, insert a slider showing milliseconds.

**Comment by ayoubkhan558** [comment]
+1

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #1021 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in the old issue will be accounted for when considering this Feature Request for our Roadmap.

Happy voting! 🙂
